libs/counterfactuals/src/lib/CounterfactualList.tsx

```diff
@@ -343,7 +359,11 @@ export class CounterfactualList extends React.Component<
               allowFreeform
               selectedKey={`${this.state.data[column.key]}`}
               options={dropdownOption.data.categoricalOptions}
-              onChange={this.updateDropdownColValue.bind(this, column.key)}
+              onChange={this.updateComboBoxColValue.bind(
+                this,
```

Comment:
can we just use arrow func instead of binding this?

Reply:
done